roles/acme/tasks/challenge/http-01/azbs.yml

```diff
@@ -0,0 +1,75 @@
+---
+- name: Install prerequisites on localhost
```

Comment:
Is this really necessary? I didn't find anything about it in the docs.

Reply:
True, i removed it.